packages/skills/carpark_client/strategy.py

```diff
@@ -47,6 +51,29 @@ def __init__(self, **kwargs) -> None:
         self._no_find_search_interval = cast(float, kwargs.pop('no_find_search_interval')) if 'no_find_search_interval' in kwargs.keys() else DEFAULT_NO_FINDSEARCH_INTERVAL
         self._max_price = kwargs.pop('max_price') if 'max_price' in kwargs.keys() else DEFAULT_MAX_PRICE
         self._max_detection_age = kwargs.pop('max_detection_age') if 'max_detection_age' in kwargs.keys() else DEFAULT_MAX_DETECTION_AGE
+
+        self.runner = CliRunner()
```

Comment:
erm not sure why you are adding this here ...I think there was a misunderstanding, please remove. The skill package stays as it is. In the docs, under `car-park.md` whenever the user is asked to modify any config we do it via the command line.